server/initialize.sql

```diff
@@ -12,13 +12,15 @@ CREATE TABLE regions(
   id   SERIAL PRIMARY KEY,
   name VARCHAR(30) NOT NULL
 );
+INSERT INTO regions(name) VALUES ('West Midlands'), ('London'), ('Scotland'), ('Manchester');
```

Comment:
Not a change, but I'll make a ticket to make a SQL file to do this just to clean up the code. So you may lose these lines when that ticket gets completed

Reply:
Yes, Georgina can remove all the chages made to this sql file.